### ISSUE TITLE
curved_tabs.css: loading burst ignores tab corners

### ISSUE THREAD

**Issue by contextnerror** [MrOtherGuy/firefox-csshacks#306]
The colored burst on page load doesn't include the bottom corners of the tabs.

https://github.com/MrOtherGuy/firefox-csshacks/assets/54487525/5ec31b27-3f7e-44bd-9425-263ecfc92b68

**Comment by MrOtherGuy** [comment]
Yeah, that's true, but sadly I don't think there any good way to fix that.

**Comment by MrOtherGuy** [comment]
Right. Wavefox creates its tab shaping effect in totally different way than what we are doing - they use mask image at tab-background whereas we have a variable sized background-image. Both have some advantages and disadvantages, and I think it would be at least pretty difficult if not outright impossible to make tab bursting effect work correctly with the way we are using.

**Comment by MrOtherGuy** [comment]
I'll keep this open in case someone figures out a nice way to do this. This *is* a bug after all, I'm just saying it might be hard to fix.
